inst/include/Rcpp/exceptions.h

```diff
@@ -243,7 +243,7 @@ namespace internal {
     inline bool is_Rcpp_eval_call(SEXP expr) {
         SEXP sys_calls_symbol = Rf_install("sys.calls");
         SEXP identity_symbol = Rf_install("identity");
-        SEXP identity_fun = Rf_findFun(identity_symbol, R_BaseEnv);
+        Shield<SEXP> identity_fun(Rf_findFun(identity_symbol, R_BaseEnv));
         SEXP tryCatch_symbol = Rf_install("tryCatch");
```

Comment:
Does `rchk` warn about the unprotected use of 'install' here?

Reply:
I was confused. No, `rchk` knows that the `Rf_install()` result is protected already, but `Rf_install()` could be an allocation (in theory) which could clean up the result of `Rf_findFun()`.